example/bwmf/block_parameter.go

```diff
@@ -0,0 +1,137 @@
+package bwmf
+
+import (
+	"fmt"
+
+	pb "github.com/taskgraph/taskgraph/example/bwmf/proto"
+	op "github.com/taskgraph/taskgraph/op"
```

Comment:
Yes. The path is ".../op", whereas the package name is "package taskgraph_op
".

On Wed, Apr 1, 2015 at 10:47 AM, Hongchao Deng <notifications@github.com>
wrote:

> In example/bwmf/block_parameter.go
> <https://github.com/taskgraph/taskgraph/pull/8#discussion_r27541308>:
>
> > @@ -0,0 +1,137 @@
> > +package bwmf
> > +
> > +import (
> > +	"fmt"
> > +
> > +	pb "github.com/taskgraph/taskgraph/example/bwmf/proto"
> > +	op "github.com/taskgraph/taskgraph/op"
>
> ??
> I mean "github.com/taskgraph/taskgraph/op" is already "op".
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/taskgraph/taskgraph/pull/8/files#r27541308>.
>